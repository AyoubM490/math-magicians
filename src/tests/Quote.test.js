import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

describe('test if Quote', () => {
  test('renders correctly', () => {
    const component = renderer
      .create(<Quote />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
