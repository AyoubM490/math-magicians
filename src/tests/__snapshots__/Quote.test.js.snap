// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if Quote renders correctly 1`] = `
<div>
  <p
    className="quoteParagraph"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: It is about understanding.
    <b>
      - William Paul Thurston
    </b>
  </p>
  <p
    className="quoteParagraph"
  >
    We will always have STEM with us. Some things will drop out of the public eye and go away, but there will always be science, engineering, and technology. And there will always, always be mathematics.
    <b>
      — Katherine Johnson, African-American mathematician
    </b>
  </p>
  <p
    className="quoteParagraph"
  >
    Pure mathematics is, in its way, the poetry of logical ideas.
    <b>
       — Albert Einstein, German theoretical physicist
    </b>
  </p>
  <p
    className="quoteParagraph"
  >
    What is mathematics? It is only a systematic effort of solving puzzles posed by nature.
    <b>
      — Shakuntala Devi
    </b>
  </p>
</div>
`;
